docs/sources/variables/url-variables.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "URL variables"
+keywords = ["grafana", "url variables", "documentation", "variables"]
+type = "docs"
++++
+
+# URL variables
+
+You can use variables in URLs to link to specific portions of your visualizations. For more information about linking, refer to [Dashboard links]({{< relref "../linking/dashboard-links.md" >}}) and [Panel links]({{< relref "../linking/panel-links.md" >}}).
+
+> **Note:** These variables changed in 6.4 so if you have an older version of Grafana please use the version picker to select
+docs for an older version of Grafana.
+
+## Typeahead suggestions
+
+Press Cmd+Space or Ctrl+Space on your keyboard to open the typeahead suggestions to more easily add variables to your URL.
+
+{{< docs-imagebox img="/img/docs/data_link_typeahead.png"  max-width= "800px" >}}
+
+## Time variables
+
+* ``__url_time_range`` - current dashboard's time range (i.e. ``?from=now-6h&to=now``)
+* ``__from`` - current dashboard's time range from value
+* ``__to`` - current dashboard's time range to value
+
+## Series variables
+
+Series specific variables are available under ``__series`` namespace:
+
+* ``__series.name`` - series name to the URL
+* ``__series.labels.<LABEL>`` - label's value to the URL. If your label contains dots use ``__series.labels["<LABEL>"]`` syntax
+
+## Field variables
+
+Field specific variables are available under ``__field`` namespace:
+
+* ``__field.name`` - field name to the URL
+
+## Value variables
```

Comment:
Available in Data links only

Reply:
Wait, what? The URL variables are only for data links? But you can add some variables to panel links.